src/ardrive.ts

```diff
@@ -64,6 +78,31 @@ export interface DriveUploadBaseCosts {
 const stubTransactionID = '0000000000000000000000000000000000000000000';
 const stubEntityID = '00000000-0000-0000-0000-000000000000';
 
+export function lastFolderRevisionFilter(
```

Comment:
Two things:
1) This function and the function below it could be 1 function if the entity types used the base type `ArFSFileOrFolderEntity`
2) Can we rely on the last element in the revisions array to be the most recent revision?

Reply:
1. Issued here: e51ef424513eb9db2f6b9fdf18a08ea236ae0a22
2. We are dealing with an array of all revisions of multiple entities. I didn't see a way of getting the last revision other than filtering. I dont know if that answers the Q